markdown/usage/tutorials/step_by_step_institutional_profile.md

```diff
@@ -0,0 +1,617 @@
+
+For example, if you're using the SLURM scheduler, you can run the following command to the walltime of any queues/partitions on your cluster with
+
+```bash
+sinfo
+```
+
+You should record the names of each `PARTITION` and the corresponding `TIMELIMIT` column. Note there may be additional resource limitations for each queue, so check your institutional cluster's documentation for any other limits.
+
+</details>
+
+## Are there any common resource or data locations on the cluster
+
+For institutions that work on common topics (e.g. genomics), you might have a centralised local copy of the Illumina `iGenome` set of reference genomes. If it exists, you can note down the paths to this directory.
+
+Furthermore, some institutions will specify a common place to store all container images (e.g. for Docker, Singularity etc) in a cache directory. This helps preventing multiple users having their own copies of the same pipeline container image. You should find out if, and where this location exists and note down the path.
+
+## Should jobs be run using a local `$TMPDIR`
+
+Some clusters require the use of a temporary _per-node_ `scratch` directory, the location of which is specified in an environmental variable `$TMPDIR`. Check if this is the case and this is set up in `$TMPDIR`
+
+## Preparation checklist
+
+To summarise, before you start writing the profile have you:
+
+- Checked you know what institutional level you will make the profile at
+- Checked permission to publicly post the profile
+- Checked which scheduler the cluster uses
+- Checked which container engine you will specify
+- Checked which maximum computational resource limits exist
+- Checked for any queue/partition names and specifications
+- Checked for locations of any common resources or cache directories
+- Checked if jobs on the cluster require use of a per-node `scratch`
+
+## Creating the profile
+
+Once you have all the information ready, we can begin writing the institutional profile! The tutorial will now go through each relevant Nextflow 'scope' block and explain what and when to put information in.
+
+Before you start consider what your institutional profile should be called. The names should be as unique and recognisable as possible - however abbreviations or acronyms are fine.
+
+For example, the name of the cluster can be used (e.g. [`phoenix`](https://github.com/nf-core/configs/blob/master/conf/phoenix.config) or [binac](https://github.com/nf-core/configs/blob/master/conf/binac.config)). In some cases an acronym of the cluster or the institute can be selected (e.g. [`SHH`](https://github.com/nf-core/configs/blob/master/conf/shh.config) or [`EVA`](https://github.com/nf-core/configs/blob/master/conf/eva.config)).
+
+Next, make sure to make your own _fork_ of [nf-core/configs](https://github.com/nf-core/configs), and make a new branch to work on. It's normally best to call the branch the name of your proposed cluster name. Clone this to your machine, and open the folder.
```

Comment:
It's not necessary to make a branch, usually people would work on their own fork of nf-core/configs once, and that's all.
In any other case, I'd recommend making a branch, in this case I think it's less important.
But as it's a good practice, I'm all for it.

Reply:
I also originally did that until recently when I got stuck doing loads of debugging and messed up master, so then couldn't pull in the latest nf-core/configs master. So I think while it's less likely, I feel it's good practise anyway (like you said) 😬